Makefile

```diff
@@ -127,9 +127,13 @@ test-integration: manifests generate fmt vet gotestsum envtest ## Run integratio
 .PHONY: kind-create
 kind-create: kind
 	kind create cluster --config hack/kind-config.yaml
+
 .PHONY: test-e2e
-test-e2e: kind docker-build install-cert-manager
-	kind load docker-image controller:latest
+test-e2e: dev-setup
+	sleep 90
```

Comment:
sleep 90 makes me sad -- ideally we could test in someway for cold start vs. already up-and-running

Reply:
I think it is smelly too. I put it because the `dev-install-controller` target fails due if some of the `dev-setup` pods aren't running yet. Can you explain more the cold start part?

Reply:
By cold start he means adding some form of "caching" behaviour. To be honest, I wouldn't like this - I think tests should start from scratch always - I.e always "cold start".

Are you able to test when all the dev-setup pods are running?https://www.ibm.com/docs/en/mvi/1.1.1?topic=environment-checking-kubernetes-services-status

Reply:
> Are you able to test when all the dev-setup pods are running?

The current test cases can run only with `dev-setup` . I could remove the `dev-install-controller` and the `sleep` commands. But I think, as this is an initial setup for E2E testing, we will later need the controller installation which requires all the pods from `dev-setup` to be up and running. 

what do you think ?